nbs/explains/minidataapi.ipynb

```diff
@@ -0,0 +1,516 @@
+{
```

Comment:
_Commented on notebook `nbs/explains/minidataapi.ipynb` Cell 6 Line 1_
```markdown
## Create database tables
```
This section should be removed - it's not part of the spec